### PR TITLE
Fix Iot DeviceUpdate Readme issues

### PR DESCRIPTION
**Pull request by v-yilinhu**
Fix issue https://github.com/Azure/azure-sdk-for-net/issues/19895

@scottaddie , @AlexGhiondea , @ericmitt , @drwill-ms, @abhipsaMisra and @jongio for notification.

### CODE REVIEW

**Comment by v-xuto** [comment]
@jongio - Who should we ping to review this PR?

File: sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ Azure Object Anchors enables an application to detect an object in the physical
 
 [Source code](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src) | [Package (NuGet)](https://www.nuget.org/packages/Azure.MixedReality.ObjectAnchors.Conversion/)
 
+
```
Comment (drwill-ms) [inline sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/README.md:7]: Why an extra newline here?

This file is not related to DeviceUpdate. Revert?

Comment (v-yilinhu) [reply sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/README.md]: I have updated the code.

**Review by drwill-ms — APPROVED** [review]

**Comment by v-xuto** [comment]
@drwill-ms  Could you help us merge this PR?

**Comment by drwill-ms** [comment]
Dupe of #21135?

**Comment by v-xuto** [comment]
Close this PR, due to  issue #19895 has been fixed.
